runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/ParDoTranslationTest.java

```diff
@@ -143,22 +143,33 @@ public void toAndFromTransformProto() throws Exception {
       inputs.putAll(parDo.getAdditionalInputs());
       PCollectionTuple output = mainInput.apply(parDo);
 
-      SdkComponents components = SdkComponents.create();
-      String transformId =
-          components.registerPTransform(
+      SdkComponents sdkComponents = SdkComponents.create();
+
+      // Encode
+      for (PValue input : inputs.values()) {
```

Comment:
Actually removing this manual registration of the transform's transitive bits does not break the test. Removed.